python/nbextension/src/widget.ts

```diff
@@ -9,15 +9,69 @@ import 'bootstrap/dist/js/bootstrap.min.js';
 import { DefaultVisualizer, MapVisualizer, StructureVisualizer } from '../../../src/index';
 import { Dataset, Settings } from '../../../src/dataset';
 
+
+class CSBaseView extends DOMWidgetView {
```

Comment:
```suggestion
class ChemiscopeBaseView extends DOMWidgetView {
```

removes the need to guess what `CS` means here